src/Microsoft.Diagnostics.Tracing.EventSource.Redist/src/Resources/Strings.resx

```diff
@@ -223,7 +223,7 @@
     <value>Attempt to define more than the maximum limit of 8 channels for a provider.</value>
   </data>
   <data name="EventSource_MismatchIdToWriteEvent" xml:space="preserve">
-    <value>Event {0} is givien event ID {1} but {2} was passed to WriteEvent.</value>
+    <value>Event {0} is given event ID {1} but {2} was passed to WriteEvent.</value>
```

Comment:
I'm having trouble with the grammar here. Should it be "has" or "is assigned" rather than "is given"?